source/modules/MEI.stringtab.xml

```diff
@@ -1,51 +1,91 @@
-
-  Licensed under the Educational Community License, Version 2.0 (the "License"); you may
-  not use this file except in compliance with the License. You may obtain a copy of the License
-  at https://opensource.org/licenses/ECL-2.0.
-  
-  Unless required by applicable law or agreed to in writing, software distributed under the
-  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
-  OF ANY KIND, either express or implied. See the License for the specific language
-  governing permissions and limitations under the License.
-  
-  This is a derivative work based on earlier versions of the schema © 2001-2006 Perry Roland
-  and the Rector and Visitors of the University of Virginia; licensed under the Educational
-  Community License version 1.0.
-  
-  CONTACT: info@music-encoding.org
+     NOTICE: Copyright (c) 2017-2023 by the Music Encoding Initiative (MEI) Board (formerly known as
+     "MEI Council").
+     
+     Licensed under the Educational Community License, Version 2.0 (the "License"); you may
+     not use this file except in compliance with the License. You may obtain a copy of the License
+     at https://opensource.org/licenses/ECL-2.0.
+     
+     Unless required by applicable law or agreed to in writing, software distributed under the
+     License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+     OF ANY KIND, either express or implied. See the License for the specific language
+     governing permissions and limitations under the License.
+     
+     This is a derivative work based on earlier versions of the schema © 2001-2006 Perry Roland
+     and the Rector and Visitors of the University of Virginia; licensed under the Educational
+     Community License version 1.0.
+     
+     CONTACT: info@music-encoding.org
 -->
 <?xml-model href="../validation/mei_odds.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <?xml-model href="../validation/mei_odds.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <specGrp xmlns="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0"
-  xmlns:sch="http://purl.oclc.org/dsdl/schematron" xml:id="module.MEI.stringtab">
+         xmlns:sch="http://purl.oclc.org/dsdl/schematron" xml:id="module.MEI.stringtab">
   <moduleSpec ident="MEI.stringtab">
     <desc xml:lang="en">Tablature component declarations.</desc>
   </moduleSpec>
+  <classSpec ident="att.course.log" module="MEI.stringtab" type="atts">
+    <desc xml:lang="en">Logical domain attributes.</desc>
+    <classes>
+      <memberOf key="att.pitched"/>
+    </classes>
+  </classSpec>
+  <classSpec ident="att.scoreDef.vis.tablature" module="MEI.stringtab" type="atts">
+    <desc xml:lang="en">Visual domain attributes for scoreDef in the tablature repertoire.</desc>
+    <classes>
+      <memberOf key="att.stringtab.vis" />
```

Comment:
Schematron error: The `<memberOf>` referencing "`att.stringtab.vis`" is broken: There is no such thing in the specs.

Reply:
There is somehting like "`att.stringtab`", "`att.stringtab.position`" and "`att.stringtab.tuning`"

Reply:
We followed the practice of other modules when attempting to implement these. Please could you kindly provide a concrete suggestion of how to proceed?